lib/mail/fields/unstructured_field.rb

```diff
@@ -175,7 +175,7 @@ def encode(value)
       value.gsub!(/\)/, '=29')
       value.gsub!(/\?/, '=3F')
       value.gsub!(/_/,  '=5F')
-      value.gsub!(/ /,  '_')
+      value.tr!(' ',  '_')
```

Comment:
This one feels out of place alongside a series of `gsub!`. What's the performance impact / is it justified?